containers/images/content-server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM quay.io/foreman/pulp
+
+RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
+    yum -y install epel-release && \
+    yum -y install python-pulp-streamer && \
```

Comment:
you can also rm -rf /var/cache/yum

Reply:
I tested that and it didn't seem to do anything more than `yum clean all`.